code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -35,8 +35,6 @@
 			/mob/living/simple_animal/hostile/syndicate/ranged,
 			/mob/living/simple_animal/hostile/syndicate/ranged/space,
 			/mob/living/simple_animal/hostile/alien/queen/large,
-			/mob/living/simple_animal/hostile/retaliate,
-			/mob/living/simple_animal/hostile/retaliate/clown,

```

Comment:
As was said, retaliate is an abstract parent and should be blocked. The blacklist doesn't catch children of listed types, so this set up allowed bats and goats while blocking clowns and abstracts.

There are a few abstracts that do spawn though, like the animated shooty gun.